npm error when blindly following the installation instructions

### What's wrong?

npm ERR! missing script: cosmos

### Steps to reproduce

Follow the instructions in the `README` file

1.  Install react-cosmos@next to get started.

`npm install react-cosmos@next`

2.  `npm run cosmos`
3.  See the error above

### Additional context

This obviously works:

`npx react-cosmos@next`

This also seems to work:

`./node_modules/.bin/cosmos`

So the problem is with not clear or, I would say, non-existing instructions how to install and use `react-cosmos` in a project if not using `npx`.

### System

Mac OS
Node & npm installed from the official package https://nodejs.org/dist/v12.13.0/node-v12.13.0.pkg

`npm --version` 6.13.0
`node --version` v12.13.0
`which node` `/usr/local/bin/node`
`which npm` `/usr/local/bin/npm`

--- a/README.md
+++ b/README.md
@@ -32,15 +32,12 @@ The [example package](example) is a useful complement to this guide.
 
 ## Getting started
 
-1\. Install via npm
+1\. Install React Cosmos
 
 ```bash
+# Using npm
 npm i --D react-cosmos@next
-```
-
-or Yarn
-
-```bash
+# or Yarn
 yarn add --dev react-cosmos@next
 ```
 
@@ -53,13 +50,18 @@ yarn add --dev react-cosmos@next
 }
 ```
 
-3\. Start dev server
+3\. Start React Cosmos
 
-Run `npm run cosmos` or `yarn cosmos` and go to [localhost:5000](http://localhost:5000) 🚀
-
----
+```bash
+# Using npm
+npm run cosmos
+# or Yarn
+yarn cosmos
+```
 
 > You may also run `npx react-cosmos@next` in your project without installing any deps.
+
+**Go to [localhost:5000](http://localhost:5000).**
 
 ## Requirements
 

--- a/README.md
+++ b/README.md
@@ -16,7 +16,7 @@ A tool for ambitious UI developers.
 
 ---
 
-**👩‍🚀 [Live demo](https://cosmos.flatris.space)**
+👩‍🚀 **[Live demo](https://cosmos.flatris.space)**
 
 [![Cosmos Next](next.png)](https://cosmos.flatris.space)
 
@@ -61,7 +61,7 @@ yarn cosmos
 
 > You may also run `npx react-cosmos@next` in your project without installing any deps.
 
-**🚀 [localhost:5000](http://localhost:5000)**
+🚀 **[localhost:5000](http://localhost:5000)**
 
 ## Requirements
 

--- a/README.md
+++ b/README.md
@@ -16,7 +16,7 @@ A tool for ambitious UI developers.
 
 ---
 
-**[Live demo.](https://cosmos.flatris.space)**
+**👩‍🚀 [Live demo](https://cosmos.flatris.space)**
 
 [![Cosmos Next](next.png)](https://cosmos.flatris.space)
 
@@ -61,7 +61,7 @@ yarn cosmos
 
 > You may also run `npx react-cosmos@next` in your project without installing any deps.
 
-**Go to [localhost:5000](http://localhost:5000).**
+**🚀 [localhost:5000](http://localhost:5000)**
 
 ## Requirements
 

--- a/README.md
+++ b/README.md
@@ -32,7 +32,9 @@ The [example package](example) is a useful complement to this guide.
 
 ## Getting started
 
-1. Install via npm
+> You may also run `npx react-cosmos@next` in your project without installing any deps.
+
+1\. Install via npm
 
 ```bash
 npm i --D react-cosmos@next
@@ -44,7 +46,7 @@ or Yarn
 yarn add --dev react-cosmos@next
 ```
 
-2. Create package.json scripts
+2\. Create package.json scripts
 
 ```diff
 "scripts": {
@@ -53,11 +55,9 @@ yarn add --dev react-cosmos@next
 }
 ```
 
-3. Start dev server
+3\. Start dev server
 
 Run `npm run cosmos` or `yarn cosmos` and go to [localhost:5000](http://localhost:5000) 🚀
-
-> You may also run `npx react-cosmos@next` in your project without installing any deps.
 
 ## Requirements
 

--- a/README.md
+++ b/README.md
@@ -32,8 +32,6 @@ The [example package](example) is a useful complement to this guide.
 
 ## Getting started
 
-> You may also run `npx react-cosmos@next` in your project without installing any deps.
-
 1\. Install via npm
 
 ```bash
@@ -58,6 +56,10 @@ yarn add --dev react-cosmos@next
 3\. Start dev server
 
 Run `npm run cosmos` or `yarn cosmos` and go to [localhost:5000](http://localhost:5000) 🚀
+
+---
+
+> You may also run `npx react-cosmos@next` in your project without installing any deps.
 
 ## Requirements
 

--- a/README.md
+++ b/README.md
@@ -16,25 +16,48 @@ A tool for ambitious UI developers.
 
 ---
 
-**Install `react-cosmos@next` to get started.**
-
-```
-npm run cosmos
-```
+**[Live demo.](https://cosmos.flatris.space)**
 
 [![Cosmos Next](next.png)](https://cosmos.flatris.space)
 
-> You may also run `npx react-cosmos@next` in your project without installing any deps.
-
-**[Live demo](https://cosmos.flatris.space)**. Keep in mind this is a static export. The dev environment, which requires a local server running, has more functionality.
+> Keep in mind this is a static export. The dev environment, which requires a local server running, has more functionality.
 
 ## Table of contents
 
-- Setup: [Requirements](#requirements) · [Config](#config) · [Compilation](#compilation) · [Webpack](#webpack)
+- Setup: [Getting started](#getting-started) · [Requirements](#requirements) · [Config](#config) · [Compilation](#compilation) · [Webpack](#webpack)
 - Usage: [Fixtures](#jsx-fixtures) · [Decorators](#decorators) · [Mocks](#declarative-mocks) · [UI controls](#ui-controlled-values) · [UI plugins](#ui-plugins) · [Static export](#static-export) · [React Native](#react-native) · [Server-side APIs](#server-side-apis)
 - FAQ: [Troubleshooting](#troubleshooting) · [Where's my old Cosmos?](#wheres-my-old-cosmos) · [Why Cosmos?](#why-cosmos) · [Credits](#credits)
 
 The [example package](example) is a useful complement to this guide.
+
+## Getting started
+
+1. Install via npm
+
+```bash
+npm i --D react-cosmos@next
+```
+
+or Yarn
+
+```bash
+yarn add --dev react-cosmos@next
+```
+
+2. Create package.json scripts
+
+```diff
+"scripts": {
++  "cosmos": "cosmos",
++  "cosmos:export": "cosmos-export"
+}
+```
+
+3. Start dev server
+
+Run `npm run cosmos` or `yarn cosmos` and go to [localhost:5000](http://localhost:5000) 🚀
+
+> You may also run `npx react-cosmos@next` in your project without installing any deps.
 
 ## Requirements
 

